### DISCUSSION TITLE
How to use --unshallow in repo.remotes.origin.pull()

### DISCUSSION THREAD

**Discussion by wangyinwen** [gitpython-developers/GitPython#1415] [Q&A] [answered]
this is my code：
```
    repo = Repo('/usr/local/sahoc/work_space/105981/codes/test_git/master')
    o = repo.remotes.origin
    o.pull('dev', '--unshallow')
```

**Comment by Byron** [comment, marked as answer]
I assume that keyword arguments are passed to git directly as described in greater detail [in the docs](https://gitpython.readthedocs.io/en/stable/tutorial.html#using-git-directly), which means `o.pull('dev', unshallow=True)` should do the job.
